src/modules/addNewChain/components/NewAddress.tsx

```diff
@@ -0,0 +1,68 @@
+import { FC } from 'react';
+
+import { Box, Button, Select, TextInput } from 'blocks';
+
+import { getSelectChains } from '../AddNewChain.utils';
+
+export type NewAddressProps = {
+  formik: any;
```

Comment:
```suggestion
  form: Type;
```

Please provide a type